qri/qri.go

```diff
@@ -168,21 +168,12 @@ func (m *Module) GetSecret(thread *skylark.Thread, _ *skylark.Builtin, args skyl
 		return skylark.None, nil
 	}
 
-	var keyx skylark.Value
-	if err := skylark.UnpackPositionalArgs("get_secret", args, kwargs, 1, &keyx); err != nil {
+	var key skylark.String
```

Comment:
If this assumption is violated, by a user calling get_secret(1) or something, does that mean UnpackPositionalArgs will return an error? That would be neat if so.